e2e/tests/kube_client.go

```diff
@@ -0,0 +1,43 @@
+//go:build e2e
+
+package e2e
+
+import (
+	"os/exec"
+)
+
+type CommandResult struct {
+	Stdout string
+	Stderr string
+	Err    error
+}
+
+var _ PocketClient = &KubeClient{}
+
+// PocketClient is a single function interface for interacting with a node.
+type PocketClient interface {
+	RunCommand(...string) (*CommandResult, error)
+}
+
+// KubeClient saves a reference to a command
+type KubeClient struct {
+	result *CommandResult // stores the result of the last command that was run.
+}
+
+// RunCommand runs a command on a KubeClient.
+func (k *KubeClient) RunCommand(args ...string) (*CommandResult, error) {
+	base := []string{"exec", "-it", "deploy/pocket-v1-cli-client", "--container", "pocket", "--", "client"}
+	args = append(base, args...)
+	cmd := exec.Command("kubectl", args...)
```

Comment:
Why are we ignoring the error?

Reply:
Which error do you mean here?

Reply:
The way I've seen this used in the past is:

```go
cmd := exec.Command("prog")
if errors.Is(cmd.Err, exec.ErrDot) {
	cmd.Err = nil
}
if err := cmd.Run(); err != nil {
	log.Fatal(err)
}
```

Source: https://pkg.go.dev/os/exec

Is that something we want to do or is `cmd.Output` capturing it?